Userland/Services/WebContent/ConnectionFromClient.cpp

```diff
@@ -490,6 +490,37 @@ Messages::WebContentServer::GetElementAttributeResponse ConnectionFromClient::ge
     return { element.get_attribute(name) };
 }
 
+Messages::WebContentServer::GetElementPropertyResponse ConnectionFromClient::get_element_property(i32 element_id, String const& name)
+{
+    auto* node = Web::DOM::Node::from_id(element_id);
+    if (!node)
+        return Optional<String> {};
+
+    if (!node->is_element())
+        return Optional<String> {};
+
+    auto& element = verify_cast<Web::DOM::Element>(*node);
+
+    auto has_property_or_error = element.has_property(name);
+    if (has_property_or_error.is_throw_completion())
+        return Optional<String> {};
+
+    if (!has_property_or_error.release_value())
+        return Optional<String> {};
```

Comment:
I think you can skip this and just do the `get()` call, and check for `undefined` in the success case. The spec points to [`8.12.2 [[GetProperty]] (P)`](https://262.ecma-international.org/5.1/#sec-8.12.2) from ES5, which I believe is [`10.1.8 [[Get]] ( P, Receiver )`](https://tc39.es/ecma262/#sec-ordinary-object-internal-methods-and-internal-slots-get-p-receiver) these days (at least they look very similar on the inside).